packages/react/src/modules/in-app-messages/components/Banner/styles.css

```diff
@@ -24,6 +24,12 @@
   font-weight: var(--knock-weight-regular);
   line-height: var(--knock-leading-3);
 }
+.knock-iam-banner__body > :first-child {
```

Comment:
Removing vertical margins at the top and the bottom to match our design, since elements like `<p>` tags usually have some margins by default.